WordPress/Classes/ViewRelated/Views/CircularImageView.swift

```diff
@@ -34,4 +37,11 @@ class CircularImageView: UIImageView {
             layer.masksToBounds = shouldRoundCorners
         }
     }
+    /// Add the custom animation on tap.
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        guard let animation = tapAnimation else {
```

Comment:
Should `super` be called here? I'm not sure that we want to intentionally disable any handling UIImageView may do with touches (although it _probably_ doesn't do anything).

Reply:
👍